examples/lighting-app/silabs/src/AppTask.cpp

```diff
@@ -166,7 +161,7 @@ void AppTask::LightActionEventHandler(AppEvent * aEvent)
 
     if (err == CHIP_NO_ERROR)
     {
-        initiated = LightMgr().InitiateAction(actor, action, NULL);
```

Comment:
That was probably your issue right here no? InitiateAction here was sometimes called in `action == LightingManager::LEVEL_ACTION` but then value can't be NULL it must be `&value`